moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -719,12 +719,12 @@ double Trajectory::getAccelerationMaxPathVelocity(double path_pos) const
       {
         if (config_deriv[j] != 0.0)
         {
-          double A_ij = config_deriv2[i] / config_deriv[i] - config_deriv2[j] / config_deriv[j];
```

Comment:
I think this code was imported from external and @mikeferguson already pointed out to not modify it to facilitate future updates. @mikeferguson, can you please confirm?

Reply:
This external repo hasn't been touched in ~6 years and cannot be actively maintained as Tobias (a friend of mine) works at Waymo which doesn't allow open source contributions. So i don't think we should have any concerns about not changing the code